.github/workflows/ci-app1.yaml

```diff
@@ -23,6 +26,13 @@ jobs:
         with:
           prefix: cluster-
           log-path: cluster/
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: aandonov/app1
+          tags: type=raw,stage-{{version}}-{{sha}}
```

Comment:
I can see no version on output, hash is there but no version...

Reply:
I changed tags to different method to separate individual portions